app/views/locations/_render_machines.html.haml

```diff
@@ -1,14 +1,15 @@
-- location_machine_xrefs.sort{|a,b| a.machine.massaged_name <=> b.machine.massaged_name}.each do |lmx|
-  %div[lmx, :machine]
-    = render :partial => 'location_machine_xrefs/list_machine', :locals => {:lmx => lmx}
-    .clear
-    - if lmx.machine.ic_eligible
-      = render :partial => 'location_machine_xrefs/ic_button', :locals => {:lmx => lmx}
-    = render :partial => 'machine_conditions/show_machine_conditions', :locals => {:lmx => lmx}
-    = render :partial => 'location_machine_xrefs/update_machine_condition', :locals => {:lmx => lmx}
-    - if user_signed_in?
-      = render :partial => 'machine_score_xrefs/add_high_score',  :locals => {:lmx => lmx}
-    = render :partial => 'machine_score_xrefs/show_high_scores', :locals => {:lmx => lmx}
-    %div.external_link_container
-      = render :partial => 'location_machine_xrefs/pintips', :locals => {:lmx => lmx}
-      = render :partial => 'location_machine_xrefs/ipdb', :locals => {:lmx => lmx}
+- location_machine_xrefs.each do |lmx|
```

Comment:
Will the results here still be sorted by massaged_name?

Reply:
yeah, I just extracted the logic from the view to the controller handling it

https://github.com/pinballmap/pbm/pull/1523/files#diff-bee6109a705a78cdfa12162d723fe642c7b7c8e53045513503b6d8e574e41b15R44